src/goto-symex/goto_symex.h

```diff
@@ -271,25 +271,41 @@ class goto_symext
 
   virtual void loop_bound_exceeded(statet &, const exprt &guard);
 
-  // function calls
+  /// Assuming \p state is currently pointing on a return instruction, assign
+  /// the value in that return to the top frame's \p return_value field.
   void return_assignment(statet &);
```

Comment:
Is this actually dead code? return-value removal should render it such.

Reply:
Indeed it looks like it, I've added a commit to replace the call to this by an UNREACHABLE